File: exercises/beecrowd/1015/1015.go
// https://judge.beecrowd.com/pt/problems/view/1015
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1, y1, x2, y2 float64

	fmt.Scanf("%f %f ", &x1, &y1)
	fmt.Scanf("%f %f", &x2, &y2)

	result := math.Sqrt(math.Pow(x2-x1, 2) + math.Pow(y2-y1, 2))

	fmt.Printf("%.4f\n", result)
}

File: exercises/beecrowd/1015/1015.java
// https://judge.beecrowd.com/pt/problems/view/1015
import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        double x1 = sc.nextDouble();
        double y1 = sc.nextDouble();
        double x2 = sc.nextDouble();
        double y2 = sc.nextDouble();

        double diffX = x2 - x1;
        double diffY = y2 - y1;
        double powX = Math.pow(diffX, 2);
        double powY = Math.pow(diffY, 2);

        double distancia = Math.sqrt(powX + powY); 
        System.out.printf("%.4f\n", distancia);

        sc.close();
    }
}
